Formula/libefi.rb

```diff
@@ -0,0 +1,27 @@
+class Libefi < Formula
+  desc "Base EFI runtime library"
+  homepage "https://github.com/rickmark/gnu-efi"
+  url "https://github.com/rickmark/gnu-efi/archive/v1.0.tar.gz"
+  sha256 "7eadece64bd593885d2d76bd5e69910ad1b246e98387df915fa15d029bf0ab45"
+
+  depends_on "gcc" => :build
+
+  def install
+    system "make"
+
+    include.install Dir["inc/*.h"]
```

Comment:
This formula needs to use `make install` to be considered for inclusion in Homebrew.